label_studio/data_import/models.py

```diff
@@ -109,6 +123,35 @@ def read_tasks_list_from_json(self):
                 raise ValidationError('Task item should be dict')
             tasks_formatted.append(task)
         return tasks_formatted
+    
+    def read_tasks_list_from_jsonl(self):
+        from .uploader import create_file_upload
+        
+        logger.debug('Read tasks list from JSONL file {}'.format(self.file.name))
+        content = self.content.splitlines()
+        tasks = []
+
+        for line in content:
+            task_data = json.loads(line)
+    
+            if 'audio_filepath' in task_data:
+                    # Get the path of the audio file
+                    audio_path = task_data.pop('audio_filepath')  # pop - to remove the original key
+                    
+                    if os.path.exists(audio_path):
+                        # Upload audio to LS file storage
+                        with open(audio_path, 'rb') as audio_file:
+                            # Create a FileUpload for the audio
+                            audio_upload = create_file_upload(self.user, self.project, SimpleUploadedFile(os.path.basename(audio_path), audio_file.read()))
```

Comment:
It most cases LS doesn't have an access to audio_path, because it's deployed on a separated machine/docker